snuba/replacers/errors_replacer.py

```diff
@@ -217,6 +217,7 @@ def process_message(
                 except Exception as e:
                     metrics.increment("errors_replacer_process_message_error")
                     logger.exception(e)
+            metrics.increment("projects_to_skip", value=len(projects_to_skip))
```

Comment:
is a counter metric what you want here?

Reply:
I think so, since I want to exact number of projects that we are skipping

Reply:
changed to `gauge`